products/customer_analytics/frontend/components/CustomerAnalyticsQueryCard.tsx

```diff
@@ -32,8 +34,8 @@ function anyValueIsNull(object: object): boolean {
     return Object.values(object).some((value) => value === null)
 }
 
-function getEmptySeriesNames(requiredSeries: object): string[] {
-    return Object.entries(requiredSeries)
+function getEmptySeriesNames(requiredSeries: object | undefined): string[] {
```

Comment:
Just a tip: `object` is basically the same as `any`. It's basically useless. You could use `Record<string, any>` if you wanna guarantee it's a JS object